# Node rate config
lineDetector: 50.0 # rate to run the Line Detector node (hz)
motorDriver: 10.0 # rate to run the Motor Driver node (hz)
